apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.IntegrationTests/StorageFixture.cs

```diff
@@ -60,6 +60,21 @@ public sealed class StorageFixture : CloudProjectFixtureBase, ICollectionFixture
         /// for 24 hours.
         /// </summary>
         public string SoftDeleteBucket => BucketPrefix + "-soft-delete";
+        /// <summary>
```

Comment:
In the fixture, we only add resources that are going to be used in multiple tests. That's not the case for any of these. You should create, and clean up, these resources on the tests that need them. You should use there the fixture's CreateBucket method, etc.